### PR TITLE
[MIG] partner_identification: Migration to 16.0

### PR DESCRIPTION
**Pull request by deu-dev**
#1353

### CODE REVIEW

File: partner_identification/tests/test_res_partner.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
+# Copyright 2017 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo.exceptions import ValidationError
+from odoo.tests import common
+
+from .fake_models import ResPartner, setup_test_model, teardown_test_model
+
+
+class TestResPartner(common.SavepointCase):
```
Comment (AnizR) [inline partner_identification/tests/test_res_partner.py]: SavepointCase is deprecated. Maybe use TransactionCase?

Comment (deu-dev) [reply partner_identification/tests/test_res_partner.py]: @AnizR changed

**Comment by rousseldenis** [comment]
@deu-dev Thanks for this.

Could you follow migration guide in order to keep commits history (Full process part) ? https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-16.0

**Comment by marielejeune** [comment]
Hi @deu-dev, since you didn't manage to fix your PR this last month, I opened a new one here: https://github.com/OCA/partner-contact/pull/1397

**Pull request closed without merging**